models/yang/annotations/sonic-neighbors-annot.yang

```diff
@@ -0,0 +1,21 @@
+module sonic-neighbors-annot {
+
```

Comment:
Any reason for having module name with neighbors instead of just neighbor?

Reply:
It returns a list of neighbors, hence 'sonic-neighbors'.

Reply:
it should be sonic-neighbor, please refer other YANG modules, e.g sonic-port, sonic-portchannel.....sonic-vlan.yang handles multiple VLAN..did we name the module as sonic-vlans??? please refer other module and define the module name accordingly.

Reply:
The name was chosen after discussing it on emails, you were also part of that loop :).  Anyway,  I have renamed it to sonic-neighbor.yang